schematic/utils/io_utils.py

```diff
@@ -11,7 +11,7 @@ def load_json(file_path):
     """
     if file_path.startswith("http"):
         with urllib.request.urlopen(file_path) as url:
-            data = json.loads(url.read().decode())
```

Comment:
Could you explain this change here for posterity?

Reply:
My bad, I removed `decode()` initially to fit the test which was giving me an error like "why do you want to decode an object that is already a string?". Later I fixed my test by adding an `encode('utf-8')` on `json.dumps()`.